sale_product_pack/tests/test_sale_product_pack.py

```diff
@@ -8,8 +8,12 @@ class TestSaleProductPack(SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        pricelist = cls.env["product.pricelist"].create({"name": "Test"})
```

Comment:
Please force the company:
```suggestion
        pricelist = cls.env["product.pricelist"].create({"name": "Test", "company_id": cls.env.company.id})
```